lib/routes/v1/index.js

```diff
@@ -85,14 +101,23 @@ router.post('/register', getSchemaValidationMw(register), asyncMw(verifyCaptcha)
 
   const pushTxData = await coSign({
     serializedTransaction: txArr,
-    signatures: [tx.s]
+    signatures: [tx.s],
+    cosignType: 'duelAuth'
   })
 
   const txRes = await pushTx(pushTxData)
   if (txRes.error) {
     throw new GenericError('Failed to push reguser transaction', txRes)
   }
 
+  const result = await userRequest({ action: 'initStakeLimit', args: [{ account: user }] })
+  if (!result.success) {
+    if (result.message.startsWith('connect ECONNREFUSED') || result.message.startsWith('ERR_GRAPE_LOOKUP_EMPTY')) {
+      throw new GenericError(`grc initStakeLimit request failed: ${result.message}`)
+    }
+    throw new GenericError(`grc initStakeLimit request failed: ${result.message}`)
```

Comment:
looks like the generic error is same here with the one thrown inside if condition above, if there's no error I think there's no need for if statement above

Reply:
Same here https://github.com/bitfinexcom/moonbeam/pull/58/files#diff-5c2b79816550450ef4ad44b52a8f3a9dR60

Reply:
will correct that, thank you